src/i18n.js

```diff
@@ -39,6 +39,7 @@ i18n
         escapeValue: false,
       },
       keySeparator: ' µ', // Deactivate the keySeparator option
+      nsSeparator: false, // Deactivate the nsSeparator option
```

Comment:
Core change

Reply:
Why ?

Reply:
In the special case of the topic zweitausbildung, we have a dropdown item 'Alle' which should be translated. This is currently translated in the general IconList component, see [line|https://github.com/geops/trafimage-maps/blob/gevos/zweitausbildung-patches/src/components/IconList/IconList.js#L72]. Other dropdown items have names that contain colons, such as "Bernina Express: Chur - ...". These items do not have to translations but are translated anyways.

In the default behavior of i18n a text containing a colon will be split. The first part is regarded as namespace, the second part is returned. As far as I know, we don't use namespaces in this application. So, I thought it would be safe to exclude them. Then we don't have to think about colons in texts which I think is a trap.